rpc-test/tests/rpc.rs

```diff
@@ -545,12 +571,7 @@ fn run_tpu_send_transaction(tpu_use_quic: bool) {
 
 #[test]
```

Comment:
Could you rename `run_tpu_send_transaction` into `test_run_tpu_send_transaction` and mark it as `test` while removing this code below?